core/.changelog.d/3503.added

```diff
@@ -1 +1,2 @@
 Added ability to cancel recovery on word count selection screen.
+
```

Comment:
this empty line should not be here.

Reply:
@obrusvit I'll remove it from the other PR also then? :)